### PR TITLE
Fixes and optimizations for low-end mobile for LW

### PR DESCRIPTION
**Pull request by aleks01010101**

### CODE REVIEW

File: ScriptableRenderPipeline/LightweightPipeline/LWRP/Editor/ShaderGraph/lightweightPBRForwardPass.template [hunk 1]
```diff
@@ -69,8 +69,8 @@ Pass
  		// We either sample GI from lightmap or SH. lightmap UV and vertex SH coefficients
 	    // are packed in lightmapUVOrVertexSH to save interpolator.
```
Comment (phi-lira) [inline ScriptableRenderPipeline/LightweightPipeline/LWRP/Editor/ShaderGraph/lightweightPBRForwardPass.template]: Very minor. Should update the commentary here as well. No need to fix it in this PR.

File: ScriptableRenderPipeline/LightweightPipeline/LWRP/Editor/ShaderGraph/lightweightPBRForwardPass.template [hunk 2]
```diff
@@ -110,10 +110,14 @@ Pass
 #ifdef _NORMALMAP
 	    inputData.normalWS = TangentToWorldNormal(Normal, WorldSpaceTangent, WorldSpaceBiTangent, WorldSpaceNormal);
 #else
+    #if !SHADER_HINT_NICE_QUALITY
+        inputData.normalWS = WorldSpaceNormal;
```
Comment (phi-lira) [inline ScriptableRenderPipeline/LightweightPipeline/LWRP/Editor/ShaderGraph/lightweightPBRForwardPass.template]: Is the vertex counterpart updated in the template as well? Can't find it.

File: ScriptableRenderPipeline/LightweightPipeline/LWRP/ShaderLibrary/LightweightPassLit.hlsl [hunk 3]
```diff
@@ -92,10 +109,10 @@ LightweightVertexOutput LitPassVertex(LightweightVertexInput v)
     // We either sample GI from lightmap or SH. lightmap UV and vertex SH coefficients
     // are packed in lightmapUVOrVertexSH to save interpolator.
```
Comment (phi-lira) [inline ScriptableRenderPipeline/LightweightPipeline/LWRP/ShaderLibrary/LightweightPassLit.hlsl]: Same here. We should update the comment.

File: ScriptableRenderPipeline/LightweightPipeline/LWRP/Editor/ShaderGraph/lightweightPBRForwardPass.template (hunk 2 above)
Comment (aleks01010101) [reply ScriptableRenderPipeline/LightweightPipeline/LWRP/Editor/ShaderGraph/lightweightPBRForwardPass.template]: It looks like it's always normalized, unless UNITY_ASSUME_UNIFORM_SCALING is defined

**Pull request merged**